hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/ContainerKeyMapperTask.java

```diff
@@ -70,6 +74,10 @@ public ContainerKeyMapperTask(ReconContainerMetadataManager
   @Override
   public Pair<String, Boolean> reprocess(OMMetadataManager omMetadataManager) {
     long omKeyCount = 0;
+    Map<ContainerKeyPrefix, Integer> containerKeyMap = new HashMap<>();
+    Map<Long, Long> containerKeyCountMap = new HashMap<>();
+    List<ContainerKeyPrefix> deletedKeyCountList = new ArrayList<>();
```

Comment:
Let's add a brief comment above each of those variables to explain how they are used (what do they store, e.g. maps from what key to what value, and how the rest of the process interacts with them).

Reply:
Thanks, I added some comments to these lines.